src/api.rs

```diff
@@ -392,14 +392,13 @@ impl Default for ChromaSamplePosition {
 #[derive(ArgEnum, Debug, Clone, Copy, PartialEq, FromPrimitive)]
 #[repr(C)]
 pub enum PixelRange {
-  Unspecified = 0,
```

Comment:
Should you set `Limited` to 0 now?

Reply:
It is 0 by default. I can be explicit if you think it is helpful.